Provide a more helpful error message when a build fails on a syntax error

```bash
make functest
err: exit status 1: stderr: build cache is required, but could not be located: GOCACHE is not defined and neither $XDG_CACHE_HOME nor $HOME are defined
```

The issue was a syntax error that the build handle trouble with:  `virtClientr` should be `virtClient`.
```diff
+                               _, err = virtClientr.CoreV1().Pods(tests.NamespaceTestDefault).Create(pod).
+                                       Expect(err).ToNot(HaveOccurred())
```

When I do

```
$ git diff
diff --git a/tests/pausing_test.go b/tests/pausing_test.go
index 79c668e40..2f112ba3e 100644
--- a/tests/pausing_test.go
+++ b/tests/pausing_test.go
@@ -414,7 +414,7 @@ var _ = Describe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com][level:com
                        By("Unpausing the VMI")
                        command = tests.NewRepeatableVirtctlCommand("unpause", "vmi", "--namespace", tests.NamespaceTestDefault, vmi.Name)
                        Expect(command()).To(Succeed(), "should successfully unpause tthe vmi")
-                       tests.WaitForVMIConditionRemovedOrFalse(virtClient, vmi, v1.VirtualMachineInstancePaused, 30)
+                       tests.WaitForVMIConditionRemovedOrFalse(virtClientr, vmi, v1.VirtualMachineInstancePaused, 30)
                        By("Verifying VMI was indeed Paused")
                        uptimeDiffAfterPausing := hostUptime() - grepGuestUptime(vmi)
```

I get 


```
/root/.cache/bazel/_bazel_root/6f347497f91c9a385dcd9294645b76e0/sandbox/processwrapper-sandbox/374/execroot/__main__/tests/pausing_test.go:417:44: undefined: virtClientr
```

Can you verify that you don't have any strange symlinks or so in your env? goimports can get confused by that.

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -50,11 +50,10 @@ http_archive(
 
 http_archive(
     name = "com_github_atlassian_bazel_tools",
-    sha256 = "e4737fd3636d23f12cd3f9880b1cfa75c1bbdd4a967852785e227f3b0ab11844",
-    strip_prefix = "bazel-tools-7d296003f478325b4a933c2b1372426d3a0926f0",
+    sha256 = "29813b426161f1f09f940e62224f4e54e5737686f2bd22146807d933fa1fa768",
+    strip_prefix = "bazel-tools-82b58b374e3b1746d6d6a58a37f7ada4400a13ce",
     urls = [
-        "https://github.com/atlassian/bazel-tools/archive/7d296003f478325b4a933c2b1372426d3a0926f0.zip",
-        "https://storage.googleapis.com/builddeps/e4737fd3636d23f12cd3f9880b1cfa75c1bbdd4a967852785e227f3b0ab11844",
+        "https://github.com/atlassian/bazel-tools/archive/82b58b374e3b1746d6d6a58a37f7ada4400a13ce.zip",
     ],
 )
 

